tools/apidoc/build-apidoc.sh

```diff
@@ -61,8 +60,8 @@ set -e
  cp "$thisdir"/*.java .
  cp "$thisdir"/*.xsl .
  sed -e 's,%API_HEADER%,All APIs,g' "$thisdir/generatetoc_header.xsl" >generatetoc.xsl
- sed -i "s/%ACS_RELEASE%/${ACS_RELEASE}/g" generatetoc.xsl
- sed -i "s/%ACS_RELEASE%/${ACS_RELEASE}/g" generatecommands.xsl
+ sed -i .bak "s/%ACS_RELEASE%/${ACS_RELEASE}/g" generatetoc.xsl
```

Comment:
What if we execute the commands at line 63 and 64 like the one in 62? Or, maybe the other way around. 

I mean, they do the same thing, but with different approaches. Therefore, it might be interesting to maintain some consistency here.

Reply:
@rafaelweingartner, not entirely sure what you mean but 62 an 63 can certainly be combined. not sure how 64 can, it is touching a different file.

Reply:
I meant the following:
All of the commands are replacing "place holders" in the same file. However, two of them are doing in line (via -i flag), whereas line 62 is replacing the entire file (not using the -i flag and redirecting the output of the `sed` to a file with the same name). I do not mind if we do in different commands the replace, but we could at least try to use the same approach in all of them. 

By approach here I mean the method, either inline or not.

Reply:
I think you are misreading the code,
 line 62 redirects to a new file, from "$thisdir/generatetoc_header.xsl" to generatetoc.xsl
 then 63 operates on the resulting generatetoc.xsl
 but 64 is operating on generatecommands.xsl

I don't think shell operations would allow redecting to input, it would empty the file before openening it for read if you tried that.

Reply:
@DaanHoogland @rafaelweingartner A (ugly) solution is running the command with the -e and move the new file to the old one:
`sed -e "s/%ACS_RELEASE%/${ACS_RELEASE}/g" newGeneratetoc.xsl && mv newGeneratetoc.xsl generatetoc.xsl;`

Reply:
@DaanHoogland on linux the `-i` must be appended with the `.bak` (`-i.bak`)

```
-i[SUFFIX], --in-place[=SUFFIX]
              edit files in place (makes backup if SUFFIX supplied)
```